nuc.oscon.2017/README.md

```diff
@@ -0,0 +1,57 @@
+# Crate.io @ OSCON 2017 in Austin, TX
+
+This is a demo that was featured on the Crate.io booth at OSCON in May 2017.
+
+## Contents
+
+A container based face tracking application that requires a Raspberry Pi and Raspberry Pi Camera to work; as well as a CrateDB cluster for the data to be inserted into. The demo will use the created Docker container to fetch images from the camera, apply a face-detection algorithm on each frame and store the results in a CrateDB cluster.
```

Comment:
"cotnainer-based"

"; as well as" -> "You will also need"

new parra at "The demo"

"frame" -> "frame,"